pkgs/sops-install-secrets/main.go

```diff
@@ -870,6 +897,21 @@ func installSecrets(args []string) error {
 		return err
 	}
 
+	if manifest.UserMode {
+		rundir, ok := os.LookupEnv("XDG_RUNTIME_DIR")
+		if !ok {
+			return fmt.Errorf("$XDG_RUNTIME_DIR is not set!")
```

Comment:
this fixes my home-manager issue:
```suggestion
		if opts.checkMode == Off && !ok {
		        // Sometimes builds run check-mode outside the users environment. Thus
		        // this env var may not be set at check time, but it must be at runtime.
			return fmt.Errorf("$XDG_RUNTIME_DIR is not set!")
```